README.md

```diff
@@ -62,100 +61,105 @@ start();
 #### Client
 
 ```js
-var Nes = require('nes');
+const Nes = require('nes');
 
 var client = new Nes.Client('ws://localhost');
-client.connect().then(() => {
-    client.request('hello', function (err, payload) {   // Can also request '/h'
-        // payload -> 'world!'
-    });
-});
+const start = async () => {
+    await client.connect();
+    const payload = await client.request('hello');  // Can also request '/h'
+    // payload -> 'world!'
+};
+start();
 ```
 
 ### Subscriptions
 
 #### Server
 
 ```js
-var Hapi = require('hapi');
-var Nes = require('nes');
+const Hapi = require('hapi');
+const Nes = require('nes');
 
-var server = new Hapi.Server();
+const server = new Hapi.Server();
 
-server.register(Nes).then(() => {
+const start = async () => {
+    await server.register(Nes);
     server.subscription('/item/{id}');
-
-    server.start(function (err) {
-        server.publish('/item/5', { id: 5, status: 'complete' });
-        server.publish('/item/6', { id: 6, status: 'initial' });
-    });
-});
+    await server.start();
+    server.publish('/item/5', { id: 5, status: 'complete' });
+    server.publish('/item/6', { id: 6, status: 'initial' });
+};
+start();
 ```
 
 #### Client
 
 ```js
-var Nes = require('nes');
-
-var client = new Nes.Client('ws://localhost');
-client.connect().then(() => {
-    var handler = function (update, flags) {
+const Nes = require('nes');
 
+const client = new Nes.Client('ws://localhost');
+const start = async () => {
```

Comment:
@hueniverse If you prefer, I could also make these IIFEs, e.g.

```js
(async () => {
    await client.connect();
})()
```
Let me know your preference.